### DISCUSSION TITLE
possibility to update other field than text ones?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2028] [Help]
![image](https://user-images.githubusercontent.com/3023526/126320396-98f00971-0c8a-48ac-8666-23cdc949cfab.png)

I have not my boolean field and my number field.

**Comment by mjashanks** [comment]
Hey @oswaldB 

Currently, a few types don't work - Boolean, Date and Relationship. I shall log this as an issue.

However, `Number`, `Text`, `Long Text` and `Options` all work for me when using the BudibaseDB. What type of datasource are you using? SQL?

**Reply by oswaldB** [reply]
I am using the BudibaseDB but I was lloking for updating a boolean.

No worry I will change it with number and set 0 as false, 1 as true.

Thank you
